sdk/src/types/block/output/account.rs

```diff
@@ -440,6 +439,12 @@ impl AccountOutput {
             potential: potential_mana,
         })
     }
+
+    /// Returns the generation amount of the output.
+    pub fn generation_amount(&self, protocol_parameters: &ProtocolParameters) -> u64 {
```

Comment:
Perhaps add something mentioning mana? Just reading this one pprobably has no idea what this is good for

Reply:
good call